packages/fuels-test-helpers/src/node_types.rs

```diff
@@ -70,115 +64,6 @@ impl Default for Config {
             block_production: Trigger::Instant,
             vm_backtrace: false,
             silent: true,
-            chain_conf: ChainConfig::local_testnet(),
-        }
-    }
-}
-
-#[cfg(feature = "fuel-core-lib")]
-impl From<Config> for fuel_core::service::Config {
-    fn from(value: Config) -> Self {
-        Self {
-            addr: value.addr,
-            max_database_cache_size: value
-                .max_database_cache_size
-                .unwrap_or(MAX_DATABASE_CACHE_SIZE),
-            database_path: match &value.database_type {
-                DbType::InMemory => Default::default(),
-                DbType::RocksDb(path) => path.clone().unwrap_or_default(),
-            },
-            database_type: value.database_type.into(),
-            utxo_validation: value.utxo_validation,
-            debug: value.debug,
-            block_production: value.block_production.into(),
-            chain_conf: value.chain_conf,
-            ..fuel_core::service::Config::local_node()
-        }
-    }
-}
-
-pub(crate) mod serde_hex {
```

Comment:
I don't remember why this was added, but looks like it's not being used anywhere

Reply:
This was probably from the time we encoded `StateConfig` entries manually. In any case, `SnapshotWriter` will take care of it all now.